simulation-construction-set-examples/simple-pendulum-with-controller/src/main/java/us/ihmc/simplePendulum/SimplePendulumSimulation.java

```diff
@@ -1,50 +1,39 @@
 package us.ihmc.simplePendulum;
 
-import us.ihmc.simulationconstructionset.SimulationConstructionSet;
-import us.ihmc.simulationconstructionset.SimulationConstructionSetParameters;
+import us.ihmc.scs2.SimulationConstructionSet2;
+import us.ihmc.scs2.simulation.robot.Robot;
 
 public class SimplePendulumSimulation
 {
-
-   // simulation step duration
    public static final double DT = 0.001;
-
-   private SimulationConstructionSet sim;
-
+   
    public SimplePendulumSimulation()
    {
+      // Setup the definition of our pendulum
+      SimplePendulumDefinition pendulumDef = new SimplePendulumDefinition();
 
-      SimplePendulumRobot robot = new SimplePendulumRobot();
-      // robot.setController(new SimplePendulumController(robot));
+      // Instantiate a SCS object
+      SimulationConstructionSet2 scs = new SimulationConstructionSet2(SimulationConstructionSet2.impulseBasedPhysicsEngineFactory());
 
-      /* Creates simulation parameters */
-      SimulationConstructionSetParameters parameters = new SimulationConstructionSetParameters();
-      // Sets data buffer to allow for this number of values for each variable to be saved.
-      parameters.setDataBufferSize(32000);
-      // Creates a new simulation
-      sim = new SimulationConstructionSet(robot, parameters);
+      // Generate a pendulum robot object according to our definition
+      Robot pendulumRobot = new Robot(pendulumDef, scs.getInertialFrame());
 
-      /*
-       * Sets the simulation to collect data every 20 simulation steps This is used to prune data so a
-       * smaller buffer is sufficient.
-       */
-      sim.setDT(DT, 20);
+      // Add the pendulum robot to the simulation
+      pendulumRobot = scs.addRobot(pendulumDef);
 
-      // Sets the ground to be visible in the 3D view
-      sim.setGroundVisible(true);
+      // Add a controller to the pendulum robot
+      SimplePendulumController penController = new SimplePendulumController(pendulumRobot.getControllerInput(), pendulumRobot.getControllerOutput());
```

Comment:
Let's avoid abbreviations, so here I'd suggest `pendulumController` instead of `penController`